ptrac.py

```diff
@@ -206,7 +206,9 @@ def ptrac_point_sample(tally, ptrac_files, pformat="ASCII", min_frac=0):
     total_voxel = X1i*X2i*X3i
     total_voxel_low = np.count_nonzero(Ss < 10)
     total_voxel_no = total_voxel-np.count_nonzero(Ss)
-    ave_hits = sum(data)/total_voxel
+    ave_hits = np.mean(Ss).round(2)  #sum(data)/total_voxel
+    from collections import Counter
+    print('Hits distribution (hits,voxels): ', Counter(Ss.flatten()).most_common(5))
```

Comment:
I can see what you tried here, but it does not seem very informative. Perhaps pyplot can provide a better solution?